Mathlib/MeasureTheory/Function/L1Space.lean

```diff
@@ -112,8 +112,6 @@ theorem hasFiniteIntegral_def {_ : MeasurableSpace α} (f : α → β) (μ : Mea
     HasFiniteIntegral f μ ↔ ((∫⁻ a, ‖f a‖₊ ∂μ) < ∞) :=
   Iff.rfl
 
-attribute [eqns hasFiniteIntegral_def] HasFiniteIntegral
```

Comment:
idem

Reply:
This one is used below. If you'd like to remove it, could you please do it yourself?